src/lib/mina_base/permissions.mli

```diff
@@ -101,3 +101,37 @@ val typ : (Checked.t, t) Typ.t
 val user_default : t
 
 val empty : t
+
+(* TODO can this be written more concisely? *)
```

Comment:
This is fine for now -- we'll get a compile error whenever we update the derivers so we'll know to go into here to fix it.

Could you add in a comment how you generated the type definition so folks who need to change it can do the same thing easily?